odt-opt-in/REPWRITERSPECS/BANNO.ODTOPTIN.V1.POW

```diff
@@ -48,6 +52,8 @@ DEFINE
 
  TRUE                         = 1
  FALSE                        = 0
+ YES                          = 1
```

Comment:
Probably doesn't matter much but just wondering on the decision to use YES/NO vs. just having a TRUE/FALSE setting.

Reply:
@rrobison44 Just seemed like a more proper way to respond to the CFG question **"When a member opts-out of OTA should the Share's OVERDRAFTTOLERANCE field be set to $0.00?"**

Reply:
@rrobison44 If you'd like it changes to TRUE/FALSE - no problems - just let me know. It'll work the same either way!

Reply:
@rrobison44 Reverted 'YES'/'NO' booleans to 'TRUE'/'FALSE' in the PowerOn code